src/main/resources/db/changelog/changes/05-create-users-roles-table.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users_roles-table
+      author: holsoni
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: role_id
+                  type: bigint
+                  constraints:
+                    nullable: false
```

Comment:
add foreign Key constraint to columns and i think both columns should be primary Key

Reply:
Yes, this 2 fields should form a composite primary key